tools/ITER_Profile_reader.py

```diff
@@ -0,0 +1,66 @@
+# This class provides routines for reading input profiles from CORSICA data files
+
+import numpy as np
+
+class ITER_Profile_reader:
+
+    def __init__(self, filename):
+        self.filename = filename
+        
+        f = open(self.filename,'r')
+        lines = f.readlines()
```

Comment:
The proper way of reading a text file in Python is as follows:
```python
with open(self.filename, 'r') as f:
    lines = f.readlines()
```
The ``with`` ensures that the file is closed when reading is finished.